core/dbt/task/compile.py

```diff
@@ -1,17 +1,17 @@
 import threading
 from typing import AbstractSet, Optional
 
-from .runnable import GraphRunnableTask
-from .base import BaseRunner
-
 from dbt.contracts.graph.manifest import WritableManifest
 from dbt.contracts.results import RunStatus, RunResult
+from dbt.events.functions import fire_event
+from dbt.events.types import CompileComplete, CompiledNode
 from dbt.exceptions import DbtInternalError, DbtRuntimeError
 from dbt.graph import ResourceTypeSelector
-from dbt.events.functions import fire_event
-from dbt.events.types import CompileComplete
-from dbt.parser.manifest import write_manifest
 from dbt.node_types import NodeType
+from dbt.parser.manifest import write_manifest, process_node
+from .base import BaseRunner
+from .runnable import GraphRunnableTask
+from ..parser.sql import SqlBlockParser
```

Comment:
Nit: Looks like we are using `from dbt.parser.xxx import xxx` above, do we want to standarize on one(Could be either one)